app/assets/javascripts/my_modules/protocols.js

```diff
@@ -99,12 +99,23 @@ function initLinkUpdate() {
   var updateBtn = modal.find(".modal-footer [data-action='submit']");
   $("[data-action='unlink'], [data-action='revert'], [data-action='update-parent'], [data-action='update-self']")
     .on('ajax:success', function(e, data) {
-      modalTitle.html(data.title);
-      modalBody.html(data.message);
-      updateBtn.text(data.btn_text);
-      modal.attr('data-url', data.url);
+      const modalBodyTemplate = (message, warning) => `<p class="message">${message}</p>
```

Comment:
Just combine message and warning in one I18n record

Reply:
Message and Warning have different styles.